### PR TITLE
Updating protocol accordingly to new NACK specs

### PR DESCRIPTION
**Pull request by Sm3r**

### CODE REVIEW

File: AP-protocol.md [hunk 1]
```diff
@@ -269,7 +269,9 @@ pub struct Ack {
 ### Nack
 If an error occurs, then a Nack is sent. A Nack can be of type:
 1. **ErrorInRouting**: If a drone receives a Message and the next hop specified in the Source Routing Header is not a neighbor of the drone, then it sends Error to the client.
+3. **DestinationIsDrone**: If a drone receives a Message which has as last hop the id of a drone.
 2. **Dropped**: If a drone receives a Message that must be dropped due to the Packet Drop Rate, then it sends Dropped to the client.
```
Comment (matteo-dinoia) [inline AP-protocol.md:273]: minor issue 3 and 2 are swapped

**Review by matteo-dinoia — COMMENTED** [review]
Seem okay.

**Comment by DiCesareDaniele** [comment]
Already done in #102

**Comment by Sm3r** [comment]
I think I can close this then

**Pull request closed without merging**